frontend/lib/norent/site.tsx

```diff
@@ -119,21 +118,20 @@ const NorentSite = React.forwardRef<HTMLDivElement, AppSiteProps>(
     );
     return (
       <>
-        <section
-          className={classnames(
-            "section",
-            "jf-above-footer-content",
-            isPrimaryPage && "is-paddingless"
-          )}
-        >
+        <section className="jf-above-footer-content">
           <span className={classnames(isPrimaryPage && "jf-white-navbar")}>
             <Navbar
               menuItemsComponent={NorentMenuItems}
               brandComponent={NorentBrand}
             />
           </span>
+          {!isPrimaryPage && (
+            <div className="jf-block-of-color-in-background" />
+          )}
           <div
-            className={classnames(!isPrimaryPage && "container")}
+            className={classnames(
+              !isPrimaryPage && "box jf-norent-builder-page"
+            )}
             ref={ref}
```

Comment:
Basically, this implementation gives the royal treatment (aka adds the snazzy letter builder background and layout) to any page but the primary pages. For now, this means that all of our `/dev` routes currently exist with this layout. Since this doesn't apply to any public facing pages (except for our 404 page, which makes sense given the "under the hood" aspect), I'm not going to prioritize adding a third style to bound the dev pages. Also, it is kinda delightful to have the nice formatting there and doesn't really limit any functionality, so maybe we'll keep it?

Reply:
Yeah I totes agree!  NOICE